lib/puppet/type/registry_key.rb

```diff
@@ -103,7 +103,7 @@ def eval_generate
 
     # get the "should" names of registry values associated with this key
     should_values = catalog.relationship_graph.direct_dependents_of(self).select {|dep| dep.type == :registry_value }.map do |reg|
-      PuppetX::Puppetlabs::Registry::RegistryValuePath.new(reg.parameter(:path).value).valuename
+      PuppetX::Puppetlabs::Registry::RegistryValuePath.new(reg.parameter(:path).value, reg.parameter(:value_name).value).valuename
```

Comment:
Does this mean it's a breaking change? As in, do users using the old syntax have to use this new syntax?